### ISSUE TITLE
Emoji icons not displaying when using penelope

### ISSUE THREAD

**Issue by stavoxnetworks** [brightio/penelope#23]
I have run into an issue where the  emoji symbols are not displaying next to the "Show Payloads" and "Quit" text when penelope is run. I did not have this issue on previous versions of Kali but now on the latest version, this occurs. 

![image](https://github.com/brightio/penelope/assets/27533789/7c703e43-8dac-49d3-a049-dd786b540058)

Any idea what may cause this behavior?

**Comment by brightio** [comment]
Hi @stavoxnetworks, I downloaded the latest Kali version (https://cdimage.kali.org/kali-2024.1/kali-linux-2024.1-virtualbox-amd64.7z) and I don't have this issue. Can you please run the following command and send me the output?
`dpkg -l fonts-noto-color-emoji`
Also what is your exact kali version?
Thank you

**Comment by stavoxnetworks** [comment]
I will get back to you this evening

**Comment by stavoxnetworks** [comment]
Sorry for the delay. Please see my version info below.

![image](https://github.com/brightio/penelope/assets/27533789/e3ac76ea-7a3a-4ed7-81de-81d192761070)

Also, the command output

![image](https://github.com/brightio/penelope/assets/27533789/4252bad6-64c3-469c-936d-afd8155d269f)

**Comment by stavoxnetworks** [comment]
Just to update this -- I kinda derived what the issue was after I saw the output for what you asked for. :) I did a reinstall of the package through APT and rebooted. 

apt install --reinstall fonts-noto-color-emoji

That took care of it. I greatly appreciate your insight. I use this tool every single day.

![image](https://github.com/brightio/penelope/assets/27533789/984de552-d787-4fbb-b2db-43f31f2dd61c)
